internal/gps/source_manager.go

```diff
@@ -188,11 +188,24 @@ func NewSourceManager(cachedir string) (*SourceMgr, error) {
 	// If it's a TemporaryError, we retry every second. Otherwise, we fail
 	// permanently.
 	//
-	// TODO: After some time, we should emit some kind of warning that we're waiting
-	// for the lockfile to be released. #534 should be address before we will do that.
+	// TODO: #534 needs to be implemented to provide a better way to log warnings,
+	// but until then we will just use stderr.
+	//
+	// #947 appears to be caused by some locking issue; as this is the most likely
```

Comment:
the note about #534 is good, but let's drop this one, as it either provides the necessary feedback for #947 on its own (and becomes out of date cruft) or isn't actually the problem (and becomes incorrect cruft).